website/src/components/policy.js

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React, {PureComponent} from 'react';
+import styled, {ThemeProvider} from 'styled-components';
+import {Container, Content, HeroImage, LogoImage, StyledCaption} from './common/styles';
+import {theme} from '../styles';
+import Header from './header';
+
+const GITHUB_PROJECT = 'https://github.com/keplergl/kepler.gl';
+const GITHUB_PROJECT_ISSUES = `${GITHUB_PROJECT}/issues`;
+
+const StyledContainer = styled(Container)`
+  background-color: black;
+  height: 100vh;
+  
+  max-width: unset;
+  
+  .kg-home__caption__description {
+    max-width: unset;
```

Comment:
can you add a max-width: 800px to this, so it doesn't expand all the way to a 2000px widescreen?